resotool/resoapp/static/resoapp/style.css

```diff
@@ -1 +1,14 @@
 @import "bootstrap/css/bootstrap.css";
+
+@media (min-width: 601px) {
```

Comment:
The 601 pixels as `min-width` is oddly specific. Where do they come from and why is this `min-width` smaller than the `max-width` below? Sorry if this is obvious, when it comes to CSS I don't know anything.

Reply:
With my understanding of CSS, the current settings in `style.css` should be correct. Those settings state that for browsers with a width of 601px or more should use the first block, giving the email table a fixed width. Browsers with a width of 600px or below should use the second, making the email table use the entire width of the given area.

The oddly specific settings are just for now. They probably need to be adjusted once we decided on how(/whether) we want to design the site as a whole.